Sources/AppStoreConnectClient/AppStoreConnectClient.swift

```diff
@@ -77,14 +79,95 @@ public struct AppStoreConnectClient {
         let result: [Release]
         switch response {
         case .ok(let okResponse):
-            result = try okResponse.body.json.data.compactMap({ Release(schema: $0) })
+            result = try okResponse.body.json.data.compactMap { Release(schema: $0) }
         default:
             let error = try handleErrors(from: response)
             throw error
         }
         return result
     }
     
+    /// Fetches a collection of TestFlight builds for a specified app from the App Store Connect API.
+    /// end-point: v1/builds
+    /// - Parameter app: The app for which to fetch builds.
+    /// - Returns: An array of `Build` objects.
+    /// - Throws: An error of type `AppStoreConnectError` if the fetch operation fails.
+    func fetchBuilds(for app: Application) async throws -> [Build] {
+        let response = try await client.builds_hyphen_get_collection(
+            query: .init(
+                filter_lbrack_app_rbrack_: [app.id],
+                sort: Operations.builds_hyphen_get_collection.Input.Query.sortPayload?.init([._hyphen_version]),
+                fields_lbrack_builds_rbrack_: Operations.builds_hyphen_get_collection.Input.Query.fields_lbrack_builds_rbrack_Payload?.init(
+                    [.version, .uploadedDate, .minOsVersion]
+                )
+            )
+        )
+        let result: [Build]
+        var responseJson: Components.Schemas.BuildsResponse
+        switch response {
+        case .ok(let okResponse):
+            responseJson = try okResponse.body.json
```

Comment:
try next
```swift
let json = try okResponse.body.json
return json.data.compactMap { Build(schema: $0) }
```
if it is valid then remove next row 105, 106, 114, 115

Reply:
It works. Changed.